app/src/components/package/package-info/Package-blog.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from "react";
+import SubDetails from "./SubDetails";
+import axios from "axios";
+import { Helmet } from "react-helmet";
+import StickyBox from "react-sticky-box";
+import Sidebar from "react-sticky-box";
+import styles from "./styles.module.css";
+import { StickyContainer, Sticky } from "react-sticky"
+import { FaUserAlt, FaCommentDots} from "react-icons/fa";
+import { AiFillHeart, AiFillCalendar, AiFillEye} from "react-icons/ai";
+import vindhyachal from './data/Vindhyachal.json'
+import top from "./data/Top-10-Places-in-Varanasi.json"
```

Comment:
I have only added Vindhyachal package with price table , not in all package because i wanted to finalize the design of the package page . I have added top to test the redirect the onClick to any other package .

Reply:
You want to remove it?